src/Lecture1.hs

```diff
@@ -77,7 +86,14 @@ and 1 is the smallest, and 7 - 1 = 6.
 Try to use local variables (either let-in or where) to implement this
 function.
 -}
-minmax x y z = error "TODO"
+minmax :: Int -> Int -> Int -> Int
+minmax x y z = max - min
+  where
+    list = [x, y, z]
+    max :: Int
+    max = head (reverse (sort list))
+    min :: Int
+    min = head (sort list)
```

Comment:
Nice idea to use `sort` 👍🏻 
Alternatively, you can use functions `maximum` and `minimum` to find the corresponding element of the list. And after that, you can remove the `import` statement and all associated warnings with it.